src/context/FoxEthProvider/FoxEthProvider.tsx

```diff
@@ -86,9 +87,11 @@ export const FoxEthProvider = ({ children }: FoxEthProviderProps) => {
     [readyToFetchLpData, accountAddress, foxLpEnabled, foxEthLpMarketData.price],
   )
 
-  const ethAccountIds = useAppSelector(state =>
-    selectAccountIdsByAssetId(state, { assetId: ethAssetId }),
-  )
+  const filter = useMemo(() => ({ assetId: ethAssetId }), [])
+
+  // TODO(gomes): deepEqualOutputFn should happen on the selector itself and give us the same reference every call but it somehow doesn't
+  // Remove the equalityFn second arg here after we figure out why
+  const ethAccountIds = useAppSelector(state => selectAccountIdsByAssetId(state, filter), isEqual)
```

Comment:
The added `isEqual` equalityFn is the guts of this PR - added it to all other consumers as a paranoia commit, but this is really the offender of the bug this PR fixes

Reply:
this is bizarre